packages/itwin/grouping-mapping-widget/api/grouping-mapping-widget.api.md

```diff
@@ -465,20 +485,23 @@ export const SearchGroupingCustomUI: ({ updateQuery, isUpdating, resetView, }: G
 // @public
 export const useGroupingMappingApiConfig: () => GroupingMappingApiConfig;
 
+// @internal (undocumented)
+export const useGroupsClient: () => IGroupsClient;
+
 // @internal
 export const useGroupsOperations: ({ mappingId, }: GroupsOperationsProps) => {
-    groups: Group[] | undefined;
+    groups: GroupMinimal[] | undefined;
     isLoading: boolean;
     refresh: () => Promise<void>;
-    onDeleteGroup: (group: Group) => Promise<void>;
-    setShowDeleteModal: Dispatch<SetStateAction<Group | undefined>>;
-    showDeleteModal: Group | undefined;
+    onDeleteGroup: (group: GroupMinimal) => Promise<void>;
+    setShowDeleteModal: Dispatch<SetStateAction<GroupMinimal | undefined>>;
```

Comment:
Can you do me a favor and replace these Dispatches with just a simple arrow function?

Reply:
For example `(showDeleteModal: GroupMinimal | undefined) => void`. Type is a bit stricter, as we are not necessarily using the previous state.

Reply:
Oh nevermind. It skipped lines, this is the doc just gathering the types. We are good.